### PR TITLE
Habib/deriv x logo ref margin trade

### PR DESCRIPTION
**Pull request by ShuvoHabib**
Changes:

We need to include Deriv x logo / references on every markets page and tabs that reference Deriv X.

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/pages/markets/components/helper/_available-platforms.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import React from 'react'
 import styled from 'styled-components'
 import PropTypes from 'prop-types'
-import { smarttrader_url } from '../../../../common/constants'
+import { smarttrader_url, derivx_url } from '../../../../common/constants'
```
Comment (sean-binary) [inline src/pages/markets/components/helper/_available-platforms.js:4]:
```suggestion
import { smarttrader_url, derivx_url } from 'common/constants'
```

Comment (sean-binary) [reply src/pages/markets/components/helper/_available-platforms.js:4]: Let's use absolute paths for this.

File: src/images/svg/deriv-x.svg [hunk 2]
```diff
@@ -0,0 +1,24 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
```
Comment (sean-binary) [inline src/images/svg/deriv-x.svg:1]: Let's minify this as well 😄

File: src/pages/markets/components/helper/_available-platforms.js [hunk 3]
```diff
@@ -108,13 +110,26 @@ const AvailablePlatforms = ({
...
                     </a>
                 )}
+                {derivX && (
+                    <a
+                        href={derivx_url}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        <StyledFlex direction="row" ai="center">
+                            <img src={DerivX} alt="DerivX" width="32" height="32" />
```
Comment (sean-binary) [inline src/pages/markets/components/helper/_available-platforms.js:120]:
```suggestion
                            <img src={DerivX} alt="Deriv X" width="32" height="32" />
```

File: src/pages/markets/components/helper/_available-platforms.js [hunk 4]
```diff
@@ -60,6 +61,7 @@ const AvailablePlatforms = ({
     dtrader,
     dbot,
     smarttrader,
+    derivX,
```
Comment (sean-binary) [inline src/pages/markets/components/helper/_available-platforms.js:64]:
```suggestion
    derivx,
```

Comment (sean-binary) [reply src/pages/markets/components/helper/_available-platforms.js:64]: Just so we're consistent with the other platforms. For the other instances as well 👍

File: src/pages/markets/components/helper/_available-platforms.js [hunk 5]
```diff
@@ -108,13 +110,26 @@ const AvailablePlatforms = ({
...
                 )}
+                {derivX && (
+                    <a
+                        href={derivx_url}
+                        target="_blank"
+                        rel="noopener noreferrer"
+                    >
+                        <StyledFlex direction="row" ai="center">
+                            <img src={DerivX} alt="DerivX" width="32" height="32" />
+                            <Text ml="0.4rem">{localize('DerivX')}</Text>
```
Comment (sean-binary) [inline src/pages/markets/components/helper/_available-platforms.js:121]:
```suggestion
                            <Text ml="0.4rem">{localize('Deriv X')}</Text>
```

**Comment by sean-binary** [comment]
The rest LGTM 👍

File: src/pages/markets/components/helper/_available-platforms.js [hunk 6]
```diff
@@ -112,7 +112,7 @@ const AvailablePlatforms = ({
                 )}
                 {derivX && (
                     <a
-                        href={derivx_url}
+                        href="/derivx"
                         target="_blank"
                         rel="noopener noreferrer"
                     >
```
Comment (sean-binary) [inline src/pages/markets/components/helper/_available-platforms.js]: You can use `LocalizedLink` here like the DBot & DTrader links.

**Review by yashim-deriv — APPROVED** [review]
LGTM~

**Review by yashim-deriv — APPROVED** [review]

File: src/pages/markets/components/helper/_available-platforms.js [hunk 7]
```diff
@@ -110,7 +112,7 @@ const AvailablePlatforms = ({
                         </StyledFlex>
                     </a>
                 )}
-                {derivx && (
+                {derivx && is_eu_country && (
```
Comment (sean-binary) [inline src/pages/markets/components/helper/_available-platforms.js:115]:
```suggestion
                {derivx && !is_eu_country && (
```

Comment (sean-binary) [reply src/pages/markets/components/helper/_available-platforms.js:115]: This should only be available for non-eu.

**Review by sean-binary — APPROVED** [review]

**Review by sean-binary — APPROVED** [review]

**Pull request merged**